antivir/scanners/scanner_clamav.py

```diff
@@ -58,5 +56,5 @@ def run(stream):
         """
         cd = ClamdAgnostic()
         scan_stream = cd.scan_stream(stream)
-        result = scan_stream.get('stream')
+        result = scan_stream.get('stream') if scan_stream else False
```

Comment:
return scan_stream.get('stream') if scan_stream else False

Reply:
I agree with you.